### DISCUSSION TITLE
OSError: Background processes not supported (fork on local runtime)

### DISCUSSION THREAD

**Discussion by askiiart** [TheLastBen/fast-stable-diffusion#352] [Q&A] [answered]
I've got a [fork](https://github.com/askiiart/universal-fast-stable-diffusion) of this which can run on a local runtime, but I'm getting this error: 
![image](https://user-images.githubusercontent.com/90910526/199603829-82f943fc-16af-4010-bf79-93d6882329c9.png)
It looks like I could just edit that file to use os.system() to run that command, though. Would there be any problems with that?
Also, where even is this code? I can't find it.

**Comment by TheLastBen** [comment, marked as answer]
add this at the beginning of the cell ` get_ipython().system = os.system`
